content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -256,14 +256,59 @@ spec:
       armory:
         terraform:
           enabled: true
+          # Remote backends is an Early Access feature. Do not use it in production. Omit this field if you do not want to use remote backends.
+          remoteBackendSupport: true
+    profiles:
+      deck:
+        # Enables the UI for the Terraform Integration stage
+        settings-local.js: |
+          window.spinnakerSettings.feature.terraform = true;
 ```
 
+This example manifest also enables the Terraform Integration UI and support for remote backends.
+
 **Halyard**
 
 ```
 hal armory terraform enable
 ```
 
+### Remote backends
+
+{{< include "experimental-feature.html" >}}
+
+This Early Access feature is available in Armory 2.AB.XY and later.
+
+The Terraform Integration supports using remote backends provided by Terraform Cloud and Terraform Enterprise.
+
+When using remote backends, keep the following in mind:
+
+* The Terraform stage must use the same Terraform version that your Terraform Cloud/Enterprise workspace runs.
+* The minimum supported Terraform version is 0.12.0.
+* If you view a `plan` action in the Terraform Cloud/Enterprise UI, the type of `plan` action that the Terraform Integration performs is a "speculative plan." For more information, see [Speculative Plans](https://www.terraform.io/docs/cloud/run/index.html#speculative-plans).
+* You cannot save and apply a plan file.
+* You cannot use Named Profiles for authentication for remote backends.
+  
+#### Enable remote backend support
+
+See either the **Halyard** or **Operator** for how to enable remote backend support. Once enabled, end users can use remote backends by configuring the Terraform Integration stage with the following:
+
+* Select a version that is 0.12.0 or later and matches the version that your Terraform Cloud/Enterprise runs.
+* Specify a remote backend.
```

Comment:
This is unclear to me. When I read this, I think "specify a remote backend on the stage" which is not needed. The Terraform applied by the stage should use a remote backend. I'm not sure how to make that distinction here, though.

Reply:
Thanks for the clarification. Hmm, how about removing the second bullet and replace it with this sentence:

```
Once a supported version is selected, the Terraform Integration can use Terraform code that references a remote backend.
```

Reply:
👍